web-ui/src/main/java/com/tutrit/webclient/controller/CarController.java

```diff
@@ -0,0 +1,69 @@
+package com.tutrit.webclient.controller;
+
+import com.tutrit.bean.Car;
+import com.tutrit.bean.CarBuilder;
+import com.tutrit.gateway.CarGateway;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.servlet.ModelAndView;
+
+import java.util.Optional;
+import java.util.function.Consumer;
+
+@Controller
+@RequestMapping("/cars")
+public class CarController {
+
+    @Autowired(required = false)
+    private CarGateway carGateway;
+
+    @GetMapping("/{id}")
+    public ModelAndView findCarById(@PathVariable String id) {
+        var mov = new ModelAndView();
+        carGateway.findCarById(id)
+                .ifPresentOrElse(addCarToModel(mov),
```

Comment:
move `addCarToModel()` on a new line to improve readability

Reply:
Done. And changed the line according to the comments below